.env.example

```diff
@@ -4,14 +4,15 @@ HOST=http://localhost:3000
 API_URL=https://api.test.faforever.com
 OAUTH_URL=https://hydra.test.faforever.com
 WP_URL=https://direct.faforever.com
+LOBBY_API_URL=http://lobby.faforever.com:4000
 CALLBACK=auth
 OAUTH_CLIENT_ID=faf-website
 OAUTH_CLIENT_SECRET=banana
 SESSION_SECRET_KEY=banana
 RECAPTCHA_SITE_KEY=6Lc4GmgaAAAAAPaFbnwuP_dBGG--SSy2Hi5UcqwK
 
 
-TOKEN_LIFESPAN=43200                       # an api token is valid for 12h
+TOKEN_LIFESPAN= 12                       # an api token is valid for 12h
```

Comment:
you cannot just change a env file variable meaning.. everyone who had this already installed will not get the change and the website will just do weird stuff.